source/triggers/groups/triggerGroup.js

```diff
@@ -15,8 +15,8 @@
 this.initialize = function() {
     this.triggers$ = [];
     this.canFire$ = [];
-    this.checkFrequency$ = 0.04 + ( Math.random() * 0.02 );
-    this.future( this.checkFrequency$ ).checkTriggers$();
+    this.checkFrequency$ = 0.08 + ( this.random * 0.02 );
+    //this.future( this.checkFrequency$ ).checkTriggers$();
```

Comment:
Technically, it should be this.random * 0,04 if the base is going to be 0.08 - but that's just me being anal. :)